OpenOversight/app/main/views.py

```diff
@@ -599,11 +599,93 @@ def add_unit():
         db.session.add(unit)
         db.session.commit()
         flash('New unit {} added to OpenOversight'.format(unit.descrip))
-        return redirect(url_for('main.get_started_labeling'))
+        return redirect(url_for('main.list_unit', unit_id=unit.id))
```

Comment:
Can I get guidance on this? Redirecting to an empty unit page feels... boring, when all they've done is type in a new unit name without any officers